x-pack/plugins/cloud_security_posture/server/create_indices/benchmark_score_mapping.ts

```diff
@@ -32,5 +32,20 @@ export const benchmarkScoreMapping: MappingTypeMapping = {
     'rule.benchmark.name': {
       type: 'keyword',
     },
+    policy_template: {
```

Comment:
This design is correct for viewing all environments, what about viewing a specific environment? would this design scale well? can you check that? 

Otherwise, the migration process my be too difficult and I would love not to have a breaking change here